plugins/fireeye_hx/icon_fireeye_hx/util/api.py

```diff
@@ -54,6 +55,23 @@ def get_host_id_from_hostname(self, params: dict) -> requests.Response:
     def get_version(self) -> requests.Response:
         return self.call_api(endpoint.version())
 
+    def check_host_quarantine_status(self, agent_id: str) -> requests.Response:
+        return self.call_api(endpoint.host_containment(agent_id)).json()
+
+    def quarantine_host(self, agent_id: str) -> bool:
+        action_endpoint = endpoint.host_containment(agent_id)
+        self.call_api(action_endpoint, "POST")
+        for i in range(180):  # pylint: disable=unused-variable
```

Comment:
For future reference, unused variables can be replaced with `_` to avoid linter warnings.